storage-operator/.dockerignore

```diff
@@ -0,0 +1,5 @@
+cmd/
```

Comment:
Not very fond we need this, but sure, for now. At some point I'd like to roll building of the whole thing in a container-based environment (indeed, maybe not using `operator-sdk build ...`), at which point this would need to be removed. Someday.

Reply:
Yes I added it to only upload the strict minimum to the docker daemon.

But that looks like premature optimization ^^ (and I even broke `operator-sdk build` when I added `build/` to the ignore => prove that it wasn't that good of an idea).

I won't be against its removal when it becomes an hindrance.